linalg/solvers.cpp

```diff
@@ -641,29 +753,29 @@ void CGSolver::Mult(const Vector &b, Vector &x) const
    }
    nom0 = nom = Dot(d, r);
    MFEM_ASSERT(IsFinite(nom), "nom = " << nom);
-   if (print_level == 1 || print_level == 3)
+   if (print_options.iterations || print_options.first_and_last)
    {
       mfem::out << "   Iteration : " << setw(3) << 0 << "  (B r, r) = "
-                << nom << (print_level == 3 ? " ...\n" : "\n");
+                << nom << (print_options.first_and_last ? " ...\n" : "\n");
    }
    Monitor(0, nom, r, x);
 
    if (nom < 0.0)
    {
-      if (print_level >= 0)
+      if (print_options.warnings)
       {
-         mfem::out << "PCG: The preconditioner is not positive definite. (Br, r) = "
+         mfem::err << "PCG: The preconditioner is not positive definite. (Br, r) = "
                    << nom << '\n';
```

Comment:
Again, I'm not sure if this was previously discussed in this PR: should we use `mfem::err` or `mfem::out` for warnings?